config/routes.rb

```diff
@@ -25,6 +24,11 @@
   }
 
   resources :proposals, only: [:index, :show] do
+    member do
+      get 'approve'
```

Comment:
Not a result of your changes, but we really should get rid of the `get` version of this url. It'd be pretty easy to send a follow up email to all of my approvers that included an image which, when loaded, approved my request.